### DISCUSSION TITLE
Feature Request: User defined aggregate functions, similar to BigQuery

### DISCUSSION THREAD

**Discussion by oh-wo** [duckdb/duckdb#14741] [Ideas]
Hi all,

Firstly thank you for such a great library.

I was curious what appetite there would be for implementing aggregate UDF in DuckDB, similar to https://cloud.google.com/bigquery/docs/user-defined-aggregates

Use case is to create statistics or overview across a number of rows quickly.

Thanks!

**Comment by dazzleduck-web** [comment]
Almost all bigdata framework such as spark, Trino, big query support this. I think the framework is already there we just need to improve documentation
